apps/client/src/common/api/project.ts

```diff
@@ -19,3 +19,25 @@ export async function getProjectData(): Promise<ProjectData> {
 export async function postProjectData(data: ProjectData): Promise<AxiosResponse<ProjectData>> {
   return axios.post(projectPath, data);
 }
+
+/**
+ * HTTP request to upload a project logo
+ */
+export async function uploadProjectLogo(file: File): Promise<AxiosResponse<ProjectLogoResponse>> {
```

Comment:
suggestion: I am not sure if we should have so many endpoints specific to the project logo

how about creating creating a patch project data endpoint?
We could add the logo here, if no logo is given, we delete existing logos.

What do you think?

Reply:
Kept the `upload` endpoint, but removed the `delete` endpoint.

When we upload the image, it saves to the `logos` folder, then returns the path to it. We can see the preview, and if we `Save`, the `projectLogo` gets updated to the logo path.